bitcoin/script/miniscript.h

```diff
@@ -841,7 +844,7 @@ struct Node {
     bool NeedsSignature() const { return GetType() << "s"_mst; }
 
     //! Do all sanity checks.
-    bool IsSafeTopLevel() const { return GetType() << "Bms"_mst && CheckOpsLimit() && CheckStackSize() && CheckTimeLocks(); }
+    bool IsSafeTopLevel() const { return GetType() << "Bms"_mst && CheckScriptSize() && CheckOpsLimit() && CheckStackSize() && CheckTimeLocks(); }
```

Comment:
I think we can change this as a validity check since this does not depend on the satisfactions and is known at parse time.